src/NuGet.Core/NuGet.Protocol.Core.v3/ReportAbuseResourceV3.cs

```diff
@@ -32,50 +39,10 @@ public ReportAbuseResourceV3()
         /// <returns>The first URL from the resource, with the URI template applied.</returns>
         public Uri GetReportAbuseUrl(string id, NuGetVersion version)
         {
-            //return Utility.ApplyPackageIdVersionToUriTemplate(_reportAbuseTemplates.First(), id, version);
-            return new Uri(String.Format(CultureInfo.InvariantCulture, "https://www.nuget.org/packages/{0}/{1}/ReportAbuse", id, version.ToNormalizedString()));
-        }
+            var url = string.Format(CultureInfo.InvariantCulture, _uriTemplate, id, version.ToNormalizedString());
```

Comment:
this doesn't account for a malformed URL coming from the server, and will fail search and other vital functionalities in the client